scripts/exgdas_efsoi.sh

```diff
@@ -0,0 +1,475 @@
+#!/bin/ksh
```

Comment:
why can't we re-use `exgdas_enkf.sh`?  JGDAS_ENKF_EFSOI should be able to set the required variables to allow using that ex script.  Similar to the enkf_fcst script.

Reply:
will reexamine